examples/unix/c11/z_queryable.c

```diff
@@ -111,6 +146,9 @@ int main(int argc, char **argv) {
 
     printf("Press CTRL-C to quit...\n");
     while (1) {
+        if ((n != 0) && (msg_nb >= n)) {
```

Comment:
```suggestion
        if (n != 0 && msg_nb >= n) {
```

Reply:
Is there a particular reason? I prefer when there is no ambiguity for operation order.

Reply:
looks very redundant, and the order of operations is strictly defined, but if you like it better, you can leave it.